src/components/modules/toolbar/blockSettings.ts

```diff
@@ -56,8 +58,8 @@ export default class BlockSettings extends Module<BlockSettingsNodes> {
    *
    * @todo remove once BlockSettings becomes standalone non-module class
    */
-  public get flipper(): Flipper {
-    return this.popover?.flipper;
+  public get flipper(): Flipper | undefined {
+    return 'flipper' in this.popover ? this.popover?.flipper : undefined;
```

Comment:
`this.popover?.flipper` should work the same here

Reply:
Nah, there will be TypeScript error. Property `flipper` exists only in PopoverDesktop and not in PopoverMobile, that's why we need to check weather the property is present